script.js

```diff
@@ -0,0 +1,15 @@
+// A function that opens and closes the mobile menu
+function toggleMenu() {
+  const menu = document.getElementById('header-flex');
+  menu.classList.toggle('active');
+}
+
+// Creating the variables to apply the toggleMenu function to
+const headerButton = document.getElementById('header-button');
+const menuOptions = document.getElementsByClassName('menu-option');
+
+// When clicked, the elements are going to execute the toggleMenu function
+headerButton.addEventListener('click', toggleMenu);
+for (let i = 0; i < menuOptions.length; i += 1) {
+  menuOptions[i].addEventListener('click', toggleMenu);
```

Comment:
- [ ] Kindly make sure to follow the font-weight for menu options.
- [ ] As you have noticed there are ( > ) on the right side of all the menu options. Kindly add all the three ( > ) at the respective place following the correct colour and size.

Your design:

<img width="458" alt="Screenshot 2022-05-10 at 1 01 41 AM" src="https://user-images.githubusercontent.com/70269507/167507634-6014fe06-351d-41ba-a6d4-480827f35e04.png">

Figma design:

<img width="315" alt="Screenshot 2022-05-10 at 1 02 59 AM" src="https://user-images.githubusercontent.com/70269507/167507648-db0225e1-6bcf-4a16-ba61-3d8e66c2b09a.png">